pkg/apparmor/Dockerfile

```diff
@@ -1,28 +1,76 @@
 # syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
 
-# Copyright (c) 2023 Zededa, Inc.
+# Copyright (c) 2023-2025 Zededa, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-FROM lfedge/eve-alpine:0f2e0da38e30753c68410727a6cc269e57ff74f2 as build
-ENV BUILD_PKGS linux-headers musl-dev musl-utils musl-libintl git gcc g++ \
-               autoconf automake libtool make flex bison bash sed gettext
+FROM --platform=${BUILDPLATFORM} lfedge/eve-alpine:0f2e0da38e30753c68410727a6cc269e57ff74f2 as builder-native-base
+ENV BUILD_PKGS="linux-headers musl-dev musl-utils musl-libintl git gcc g++ \
+               autoconf automake libtool make flex bison bash sed gettext"
 ENV PKGS alpine-baselayout
 RUN eve-alpine-deploy.sh
 
+FROM --platform=${BUILDPLATFORM} lfedge/eve-cross-compilers:fb809cfb1909752acb563e0b77cd3799534bce64 AS cross-compilers
+
+FROM builder-native-base as builder-cross-base
+COPY --from=cross-compilers /packages /packages
```

Comment:
Can you not just move `FROM .... AS cross-compilers` to before line 6 (previous `FROM`), and then `COPY` these into `builder-native-base`? Or is this a different platform?

This needs comments to make it clear what each one is doing, as well as how the whole flow works. I mostly get what you are doing, but miss some of the steps.